### PR TITLE
Improves JS Number type description

### LINKED ISSUE #14288: Issue with "JavaScript data types and data structures": Number type
MDN URL to Section [This is the specific headline](
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#number_type)

First of all thanks. I just wonder the wording could be improved here, or I am misunderstanding it.

#### What information was incorrect, unhelpful, or incomplete?

> The Number type is a [double-precision 64-bit binary format IEEE 754 value](https://en.wikipedia.org/wiki/Double_precision_floating-point_format) (numbers between -(2^53 − 1) and 2^53 − 1)

Do you mean integers there? 

The IEEE754 for 64bits numbers assigns  53 bits for the base + 11 for the exponent. So it's a more general, useful built in way to represent rational numbers. 

Not sure what that range is trying to explain. 

-----------

A nice calculation

This is just a random stuff in case you like numbers sorry

IEEE754 is used for both rational numbers and integers. Because 2^10 is approx 10^3, you have about 15 decimals in every number. From Wikipedia it says 15-17 range.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/javascript/data_structures`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/javascript/data_structures/index.md
* Last commit: https://github.com/mdn/content/commit/181dac2884196d07f0d11f781dd719d23b95a49f
* Document last modified: 2022-03-20T06:13:34.000Z

</details>

### PR DESCRIPTION
**Pull request by NiedziolkaMichal**
#### Summary
I added info about min and max value, described NaN in two sentences and corrected sentence which caused misunderstanding in #14288.

#### Related issues

Fixes #14288 

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by santimirandarp** [comment]
There are typos in the single paragraph you added
 
I don't think this is in the range is it? 2^-1074

**Comment by NiedziolkaMichal** [comment]
2^-1074 is correct. [Specification ](https://tc39.es/ecma262/multipage/numbers-and-dates.html#sec-number.min_value) mentiones 5x10^-324, which is [very close](https://www.wolframalpha.com/input?i=+2%5E-1074) to 2^-1074. That same number is also mentioned in [MIN_VALUE](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MIN_VALUE#description).

File: files/en-us/web/javascript/data_structures/index.md [hunk 1]
```diff
@@ -56,9 +56,9 @@ A variable that has not been assigned a value has the value `undefined`. See {{j
 
 ECMAScript has two built-in numeric types: **Number** and **BigInt** (see below).
 
-The Number type is a [double-precision 64-bit binary format IEEE 754 value](https://en.wikipedia.org/wiki/Double_precision_floating-point_format) (numbers between -(2^53 − 1) and 2^53 − 1). In addition to representing floating-point numbers, the number type has three symbolic values: `+Infinity`, `-Infinity`, and {{jsxref("NaN")}} ("**N**ot a **N**umber").
+The Number type is a [double-precision 64-bit binary format IEEE 754 value](https://en.wikipedia.org/wiki/Double_precision_floating-point_format). It is capable of storing floating-point numbers between 2^-1074 and 2^1024, but integers can be safely used only in range -(2^53 − 1) to 2^53 − 1. Values outside of {{jsxref("Number.MIN_VALUE")}} and {{jsxref("Number.MAX_VALUE")}} range, are automatically converted to either `+Infinity` or `-Infinity`. They behave similary to mathematical infinity, but with some slight differences; see {{jsxref("Number.POSITIVE_INFINITY")}} for details.
```
Comment (sideshowbarker) [inline files/en-us/web/javascript/data_structures/index.md:59]:
```suggestion
The Number type is a [double-precision 64-bit binary format IEEE 754 value](https://en.wikipedia.org/wiki/Double_precision_floating-point_format). It is capable of storing floating-point numbers between 2^-1074 and 2^1024, but only integers in the range -(2^53 − 1) to 2^53 − 1. Values outside of the range from {{jsxref("Number.MIN_VALUE")}} and {{jsxref("Number.MAX_VALUE")}} are automatically converted to either `+Infinity` or `-Infinity`. They behave similarly to mathematical infinity, but with some slight differences; see {{jsxref("Number.POSITIVE_INFINITY")}} for details.
```

File: files/en-us/web/javascript/data_structures/index.md [hunk 2]
```diff
@@ -56,9 +56,9 @@ A variable that has not been assigned a value has the value `undefined`. See {{j
 
 ECMAScript has two built-in numeric types: **Number** and **BigInt** (see below).
 
-The Number type is a [double-precision 64-bit binary format IEEE 754 value](https://en.wikipedia.org/wiki/Double_precision_floating-point_format) (numbers between -(2^53 − 1) and 2^53 − 1). In addition to representing floating-point numbers, the number type has three symbolic values: `+Infinity`, `-Infinity`, and {{jsxref("NaN")}} ("**N**ot a **N**umber").
+The Number type is a [double-precision 64-bit binary format IEEE 754 value](https://en.wikipedia.org/wiki/Double_precision_floating-point_format). It is capable of storing floating-point numbers between 2^-1074 and 2^1024, but only integers in the range -(2^53 − 1) to 2^53 − 1. Values outside of the range from {{jsxref("Number.MIN_VALUE")}} and {{jsxref("Number.MAX_VALUE")}} are automatically converted to either `+Infinity` or `-Infinity`. They behave similarly to mathematical infinity, but with some slight differences; see {{jsxref("Number.POSITIVE_INFINITY")}} for details.
```
Comment (sideshowbarker) [inline files/en-us/web/javascript/data_structures/index.md:59]:
```suggestion
The Number type is a [double-precision 64-bit binary format IEEE 754 value](https://en.wikipedia.org/wiki/Double_precision_floating-point_format). It is capable of storing floating-point numbers between 2^-1074 and 2^1024, but only integers in the range -(2^53 − 1) to 2^53 − 1. Values outside of the range from {{jsxref("Number.MIN_VALUE")}} and {{jsxref("Number.MAX_VALUE")}} are automatically converted to either `+Infinity` or `-Infinity`, which behave similarly to mathematical infinity, but with some slight differences; see {{jsxref("Number.POSITIVE_INFINITY")}} for details.
```

File: files/en-us/web/javascript/data_structures/index.md [hunk 3]
```diff
@@ -56,9 +56,9 @@ A variable that has not been assigned a value has the value `undefined`. See {{j
 
 ECMAScript has two built-in numeric types: **Number** and **BigInt** (see below).
```
Comment (sideshowbarker) [inline files/en-us/web/javascript/data_structures/index.md:57]:
```suggestion
ECMAScript has two built-in numeric types: **Number** and **BigInt**:
```

**Comment by NiedziolkaMichal** [comment]
@sideshowbarker Thank you for the fixes.

**Review by sideshowbarker — APPROVED** [review]

**Comment by sideshowbarker** [comment]
> @sideshowbarker Thank you for the fixes.

Thanks much for writing up the clarifications here

**Pull request merged**